apps/prairielearn/src/schemas/schemas/infoQuestion.json

```diff
@@ -111,11 +127,14 @@
         },
         "enableNetworking": {
           "description": "Whether the grading containers should have network access. Access is disabled by default.",
-          "type": "boolean"
+          "type": "boolean",
+          "default": false
         },
         "environment": {
           "description": "Environment variables to set inside the grading container.",
-          "type": "object"
+          "type": "object",
+          "additionalProperties": {},
+          "properties": {}
```

Comment:
In other schemas, we have this instead:

```
          "additionalProperties": {
            "type": "string"
          }
```

Is there a reason we aren't doing that here too?

Reply:
Addressed in 57735c1 and #11789.